languages/en.ini

```diff
@@ -1014,6 +1029,14 @@ page_num = "Page %%page%%"
 page_prev = "Go to Previous Page"
 page_reload_on_deselect_hint = "Page will reload when a filter is removed."
 page_reload_on_select_hint = "Page will reload when a filter is selected or excluded."
+Pages = "Pages"
```

Comment:
"Pages" seems like a very vague translation key -- I'm also not sure how it differs from the "Notifications Pages" key. Maybe it would be a good idea to prefix all/most of the new strings with `notifications_` -- e.g. `notifications_pages_short = "Pages"` and `notifications_pages = "Notifications Pages"`, etc.